ui/src/app.js

```diff
@@ -272,16 +272,24 @@ function processData (data) {
 }
 
 function translate (data, fetchArgs) {
+  if (data === "")
```

Comment:
Did you figure out what's causing data to be empty?

Reply:
A lot of elements on custom requirements screens. Those are populated dynamically (after initial translation).

We should (probably) remove i18n tags form those elements (as operator can't translate them), but I put this piece of code just to make `translate` function a little bit safer and faster in case that void tags appear again in future.

Reply:
But, yeah this if should be removed.